pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -453,7 +453,8 @@ type datastoreConfAndCtx struct {
 }
 
 // prepareDatastoresList() - lookup datastore by its UUID and construct a context.
-//                           returns null and an error if any of the operations fail
+//
```

Comment:
I appreciate reducing or removing the indentation on subsequent lines, but why add an empty 2nd comment line here and in the comments below?

Reply:
That actually comes from 'go fmt':
```
christoph@ponyhof ~/p/e/p/p/c/downloader (master)> go version
go version go1.19.6 linux/amd64
christoph@ponyhof ~/p/e/p/p/c/downloader (master)> go fmt downloader.go
downloader.go
christoph@ponyhof ~/p/e/p/p/c/downloader (master)> git diff downloader.go 
diff --git a/pkg/pillar/cmd/downloader/downloader.go b/pkg/pillar/cmd/downloader/downloader.go
index 33339edce..a04705033 100644
--- a/pkg/pillar/cmd/downloader/downloader.go
+++ b/pkg/pillar/cmd/downloader/downloader.go
@@ -453,7 +453,8 @@ type datastoreConfAndCtx struct {
 }
 
 // prepareDatastoresList() - lookup datastore by its UUID and construct a context.
-//                           returns null and an error if any of the operations fail
+//
+//     returns null and an error if any of the operations fail
 func prepareDatastoresList(ctx *downloaderContext, dlconf types.DownloaderConfig,
        dsids []uuid.UUID) ([]datastoreConfAndCtx, error) {
```